modules/ROOT/pages/_partials/include-insights-charts.adoc

```diff
@@ -0,0 +1,386 @@
+//provides includes for reference sections in anypoint-insights.adoc 
+
+//-----------------------//
+// APP DETAIL HEADER INFO:
+//tag::app-detail-header[]
+Type::
+Mule application type (*Application*)
+
+Status::
+Status code that indicates whether this Mule app is running or in some other xref:runtime-manager::managing-deployed-applications.adoc#status_states[state]
++
+This code comes from Runtime Manager.
+
+URL:: 
+URL of this deployed Mule app
+
+//TODO: Runtime -> Runtime Version (SATHISH)
+Runtime::
+Mule runtime engine version of this Mule app
+
+Replicas::
+Virtual core fraction and count for this Mule app, for example, *0.1 vCores x 1* (= 10% of 1 vCore)
+
+Last Updated::
+Date-time stamp for the last redeployment of this Mule app
++
+This stamp comes from Runtime Manager data.
+//end::app-detail-header[]
+//
+//
+// APP DETAIL: Overview tab
+//tag::app-detail-overview[]
+
+//  shared metric in Overview tab
+include::partial$include-insights-charts.adoc[tag="average-latency"]
+include::partial$include-insights-charts.adoc[tag="p95-latency"]
+include::partial$include-insights-charts.adoc[tag="total-requests"]
+
+//TODO: Messages -> Message (SATHISH)
+Mule Messages Volume::
+Total Mule message count for this app
++
+For information about Mule messages, see xref:mule-runtime::about-mule-event.adoc[].
+
+//  shared metric in Overview tab
+include::partial$include-insights-charts.adoc[tag="request-latency"]
+
+Inbound Request Volume::
+Inbound request count at specific time points
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+//  shared metric in Overview tab
+Failed Inbound Request Volume::
+Count of unsuccessful inbound requests at specific time points
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+//  shared metric in Overview tab
+Mule Messages::
+Mule message count for this Mule app at specific time points
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
++
+For information about Mule messages, see xref:mule-runtime::about-mule-event.adoc[].
+//end::app-detail-overview[]
+//---------------------------------------------------------------------------------//
+//
+//---------------------------------------------------------------------------------//
+// APP DETAIL REQUESTS: Requests tab
+//tag::app-detail-requests[]
+Inbound Request Volume::
+Inbound request count total, which you can group by response time or protocol 
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Outbound Request Volume::
+Outbound request count total, which you can group by response time, protocol, or by the 20 hosts (such as an Object Store or a `cloudhub.io` address) with the most outbound requests 
```

Comment:
How about `Total number of outbound requests made by the application in the selected time duration, with options to group by protocol, response type or by Top 20 hosts`? I am not sure if citing example of name of `host` in documentation is very helpful (it is very customer specific).

Reply:
Customers ask us for examples, so I think it can be helpful to describe what's meant.